### DISCUSSION TITLE
[359550] 彩虹六号:围攻

### DISCUSSION THREAD

**Discussion by IsaacKey** [pjy612/SteamManifestCache#126] [Share]
`appid:359550`
keys:
```
359550;2e65ef955e06496af51ea976d8059098be9889086fc44246961410940ba652c6
359551;b13d0908374e12054e73230c57da5d674dfe45cffe6ba7e11b3f1a8b155938d0
377234;c3451634ddea46eae104112a888d041e515f19dd2a838bc5bca500bd8e721d0a
377237;55e7ea1db9a23f549c35553adb88ac3f97c1fc5f649df1515f5fa1dde7f501a6
``` 
内测码:F3A43A0D5CCD79AC
[359551_7924714545170578579.zip](https://github.com/pjy612/SteamManifestCache/files/13758296/359551_7924714545170578579.zip)
[377234_6056122284734366189.zip](https://github.com/pjy612/SteamManifestCache/files/13758482/377234_6056122284734366189.zip)
[377237_7221654846420332492.zip](https://github.com/pjy612/SteamManifestCache/files/13758483/377237_7221654846420332492.zip)

邮箱: isaackey0@gmail.com

**Comment by pjy612** [comment]
密钥请和depot 对应，然后 用 \`\`\` 围起来 方便别人复制

**Reply by IsaacKey** [reply]
这样?之前没用过GitHub不是很熟悉

**Reply by pjy612** [reply]
格式对了，但是密钥有误或不全
https://steamdb.info/app/359550/depots/

**Reply by IsaacKey** [reply]
这回应该全了

**Reply by pjy612** [reply]
查收

**Comment by Malik506-dev** [comment]
Can I Play Online also ?

**Reply by pjy612** [reply]
no,just download

**Reply by Malik506-dev** [reply]
after download can i play online too or only ???

**Reply by pjy612** [reply]
No, I think you need to buy or find an account.

**Comment by Malik506-dev** [comment]
Where to find

**Reply by ZEERDEER** [reply]
Just like you find food.
